lib/auth/oidc.go

```diff
@@ -314,6 +314,18 @@ func (a *Server) validateOIDCAuthCallback(q url.Values) (*oidcAuthResponse, erro
 
 	log.Debugf("OIDC claims: %v.", re.claims)
 
+	emailVerified, hasEmailVerifiedClaim, err := claims.StringClaim("email_verified")
```

Comment:
Is it ok to use `StringClaim` to fetch `email_verified`? As for the OIDC specification, [it is a `boolean` value](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) ([we can see it on the Okta example](https://developer.okta.com/docs/reference/api/oidc/#id-token-payload)), and the [`StringClaim` tries to cast the value into a `string`](https://github.com/gravitational/go-oidc/blob/master/jose/claims.go#L24), returning an error otherwise.

Reply:
I didn't run into any errors. That said I've handling for both types because I don't imagine there's IdPs that do both.